src/native/linux_x11/glx.rs

```diff
@@ -311,16 +311,29 @@ impl Glx {
         }
 
         // _sapp_x11_grab_error_handler(libx11);
-        let attribs: [libc::c_int; 8] = [
-            GLX_CONTEXT_MAJOR_VERSION_ARB,
-            2,
-            GLX_CONTEXT_MINOR_VERSION_ARB,
-            1,
-            GLX_CONTEXT_FLAGS_ARB,
-            GLX_CONTEXT_CORE_PROFILE_BIT_ARB,
-            0,
-            0,
-        ];
+        let attribs: [libc::c_int; 8] = if(cfg!(feature="opengl3")){
```

Comment:
opengl will create the highest possible context version despite those number, so its totally fine to leave 2.1 here

Reply:
A system that only has 2.1 won't be able to use textures with the RGBA16 format. So this way opengl3 will always be loaded, no? Or will it also load lower versions and thus a check needs to happen to make sure that opengl3 got loaded?

Reply:
The idea here - if the system only has 2.1 - let it create it's 2.1 context, but turn on all the compatibility code to make the app somewhat working on 2.1. 

If we can't do software emulation or other techniques - let the user to decide if the app should print a nice panic message or do some workarounds for missing features on runtime.